.github/actions/test-notify-slack/action.yml

```
@@ -0,0 +1,22 @@
+---
+
+name: 'Test notify slack action'
+description: 'Runs validation against the notify-slack action'
+
+runs:
+  using: "composite"
+  steps:
+    - name: 'Checkout'
+      uses: actions/checkout@v3
+
+    - name: 'Post message to slack'
+      id: slack
+      uses: ./actions/notify-slack
+      with:
+        application: github-actions
+        environment: staging
+        status: success
+        type: deployment
+        version: v1.0.56
+        template: .github/actions/test-notify-slack/message.json.j2
+        slack-webhook-url: ${{ inputs.slack-webhook-url }}
```

This workflow has no inputs.

```suggestion
        slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
```